main/src/org/destinationsol/TextAlignment.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016 MovingBlocks
```

Comment:
It's 2017, my good sir :)

Just a minor nitpick, but bump up the year whenever you work on a file.